backend/src/main/java/de/lindatroesken/backend/model/MangelEntity.java

```diff
@@ -0,0 +1,36 @@
+package de.lindatroesken.backend.model;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.time.Instant;
+
+@Entity
+@Table(name = "mangel")
+@Setter
+@Getter
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@NoArgsConstructor
+public class MangelEntity {
+    @Id
+    @GeneratedValue
+    @Column(name = "id", nullable = false)
+    private Long id;
+
+    @Column(name="description", columnDefinition="TEXT")
+    private String description;
+
+    @Column(name = "date_noticed")
+    private Instant dateNoticed;
```

Comment:
ich weiss es nicht genau, aber wie funktioniert Instant? Ich dachte du würdest da eher wann die Person es bemerkt hat, nicht wann ers in die App eingetragen hat. Aber vlt verstehe ich Instant falsch (da gibts bspw Date sonst dafür)

Reply:
Mir hatte Piers den Tipp gegeben Instant als Typ für das Datum zu verwenden, anstelle von java.util, da das veraltet ist. Es gibt auch noch etwas wie LocalDate, aber ich schaue mir das mit dem Datum noch mal genauer an.